lib/convict.js

```diff
@@ -401,7 +407,7 @@ var convict = function convict(def) {
       var self = this;
       if (!Array.isArray(paths)) paths = [paths];
       paths.forEach(function(path) {
-        overlay(cjson.load(path), self._instance, self._schema);
+        overlay(loadJSON(path), self._instance, self._schema);
```

Comment:
Yes cjson uses sync everywhere, but on this particular line it's easy to use async. So if we can easily free ourselves from some sync calls in the process of moving from cjson to json5, let's do it. But OK that can be done in another PR if there are problems doing so. Let's not stop progress!